components/Header/Header.tsx

```diff
@@ -0,0 +1,56 @@
+import ToggleTheme from "./ToggleTheme";
+import { Avatar } from "@mui/material";
+import { auth } from "../../firebase";
+import Logo from "../Logo"
+import Link from 'next/link'
+import SearchBar from './SearchBar';
+import SettingsButton from './SettingsButton'
+import AppsButton from './AppsButton';
+import NavLinks from './NavLinks';
+
+const Header = () => {
+  return (
+    <div>
+        {/* Top: Toolbar */}
+        <div 
+        className='grid grid-cols-2 md:grid-cols-3 gap-y-2 grid-flow-row-dense w-full 
+        py-5 px-3 justify-items-stretch items-center bg-white dark:bg-neutralDark-500'>
+            {/* Left: Logo */}
+            <div 
+            className="flex items-center justify-self-start cursor-pointer md:mr-auto
+            text-black dark:text-neutralDark-50">
+                <Link href="/" passHref>
+                    <a><Logo fill="currentColor"/></a>
+                </Link>
+            </div>
+        
+            {/* Center: Search */}
+            <div 
+            className="flex items-center justify-center col-span-2 md:col-span-1 md:justify-items-start">
+                <SearchBar placeholder="What's your question?" />
```

Comment:
Or "Just ask" to make it fancy.
In the design they have "Search for an answer".
I think "What's your question?" or "Just ask" work better.

Reply:
The other thing I noticed is the look on smaller screens. Not sure what's the best way to show it. We can prob. leave it as is for now and ask designers.

Reply:
Every design I see in Figma's [engineering ready design](https://www.figma.com/file/Tur7BcWjVW9F6xfFY0jsRG/Oogway?node-id=233%3A2480) have "What's your question?" as the placeholder.  It's an easy change if need be.

We can even refactor it as a prop to handle more complex, conditional cases if need be. That's what I have for now and I'm leaving it.

Reply:
I'm leaving it is for now since everything is functionally operable on every mobile device that firefox developer includes (14 OSes). See comments below.